client/src/components/NavigationTitle.vue

```diff
@@ -1,17 +1,20 @@
 <script>
 export default {
-  name: "NavigationTitle",
+  name: 'NavigationTitle',
   computed: {
     version() {
       return process.env.VERSION;
-    }
-  }
+    },
+  },
 };
 </script>
 
 <template>
   <v-toolbar-title class="my-title mx-5">
-    <v-tooltip open-delay="1000" bottom>
+    <v-tooltip
+      open-delay="1000"
+      bottom
+    >
```

Comment:
1000ms seems a bit excessive for a tool tip unless it is expected that it isn't used most of the time.  It is only a version number.

Additionally have we considered linking the Logo to the current root of the site.  Most apps utilize the logo as a de facto home button.